src/bitcoincore.rs

```diff
@@ -173,6 +183,25 @@ impl BitcoinRpcApi for BitcoinCoreClient {
         ))
     }
 
+    fn get_block_verbose(&self, header_hash: &BlockHash) -> Result<Result<VerboseBlock, RpcError>, HTTPError>  {
+        let req = &RpcRequest::new2(
+            JsonRpcVersion::V1,
+            "42",
+            "getblock",
+            header_hash,
+            2
+        );
+        println!("{}", serde_json::to_string(req).unwrap());
```

Comment:
There is a log statement in the `jsonrpc` client that you can enable in order to see the communication between the client and the node.
Just set the env variable: `RUST_LOG=jsonrpc_client=debug`